### PR TITLE
Update plugin for 2.10

### PR DESCRIPTION
**Pull request by dgarros**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -8,8 +8,9 @@ env:
 # Each version of NetBox listed here must have a corresponding directory/configuration file 
 # under development/netbox_<NETBOX_VER>/configuration.py
   matrix:
-  - NETBOX_VER=v2.8.3
-  - NETBOX_VER=master
+  - NETBOX_VER=v2.8.9
+  - NETBOX_VER=v2.9.9
```
Comment (glennmatthews) [inline .travis.yml:12]: Not 2.9.11?

File: netbox_onboarding/views.py [hunk 2]
```diff
@@ -27,52 +25,74 @@
...
         permission_required = "netbox_onboarding.delete_onboardingtask"
 
-    class ReleaseMixinOnboardingTaskFeedBulkImportView(PermissionRequiredMixin):
+    class ReleaseMixinOnboardingTaskFeedBulkImportView(PermissionRequiredMixin, BulkImportView):
         """Release Mixin View for bulk-importing a CSV file to create OnboardingTasks."""
 
         permission_required = "netbox_onboarding.add_onboardingtask"
 
 
+elif NETBOX_RELEASE_CURRENT < NETBOX_RELEASE_29 and NETBOX_RELEASE_CURRENT > NETBOX_RELEASE_210:
```
Comment (glennmatthews) [inline netbox_onboarding/views.py:60]: Less than 2.9 and greater than 2.10?

File: .travis.yml (hunk 1 above)
Comment (dgarros) [reply .travis.yml]: Fixed

File: netbox_onboarding/views.py (hunk 2 above)
Comment (dgarros) [reply netbox_onboarding/views.py]: Fixed, thanks

**Review by glennmatthews — APPROVED** [review]

**Comment by mzbroch** [comment]
I am still seeing two issues with the current release (tested under 2.10.2):
- Change Log for onboarding task does not work (passes in unit tests)
```
Reverse for 'onboardingtask_changelog' with keyword arguments '{'pk': ''}' not found. 1 pattern(s) tried: ['plugins/onboarding/(?P<pk>[0-9]+)/changelog/$']
```
- Plugin icon buttons does not work (shows lines as colours only, no icons)

**Comment by mzbroch** [comment]
In 41e3b1a I added two OnboardingTask template files that are NetBox version dependent - this is for NetBox 2.8 and 2.9 support - other potential solution could be through registering django template tags and re-using a common variable name inside a template, but although technically more powerful, complexity was increased.

**Review by mzbroch — APPROVED** [review]

**Comment by dgarros** [comment]
Looks good to me, Glenn is out this week and since there is a growing number of requests for 2.10 support I propose we merge it and publish a new release ASAP

**Pull request merged**